### ISSUE TITLE
a11y issue: Unable to tab into Youtube modal window/iframe and stop, pause, play.

### ISSUE THREAD

**Issue by cymruhangu** [cymruhangu/thinkful-tube#6]

**Comment by cymruhangu** [comment]
I have modified the code so that when thumbnail clicked the iframe has focus and you are able to tab through all of the native Youtube controls.   Talk with Marc on Tuesday to see if this issue could be considered closed.

**Comment by cymruhangu** [comment]
This feature works both when pressing enter to open in the iframe or control option spacebar.
